modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,145 @@
 
 [discrete]
-== stable-{product-version} Channel
+== stable-{product-version} channel
 
-Like the fast-{product-version} channel, the stable-{product-version} channel will
-only contain releases that have been declared general availability are therefore
-fully supported. However the stable-{product-version} channel will gradually roll out
-releases to customers based on data from our SRE teams, support services, and
+Like the `fast-{product-version}` channel, the `stable-{product-version}` channel
+contains only releases that have general availability and are
+fully supported. While the `fast-{product-version}` channel contains releases as soon
+as they are released, releases are added to the the `stable-{product-version}` channel
+after data from Red Hat SRE teams, Red Hat support services, and
 pre-production and production environments that participate in our connected customer
-program, rather than being immediately available as they are in the fast-{product-version}
-channel. For patch and CVE fixes this delay can range from several hours to a day
+program confirm that they are stable. This delay can range from several hours to a day
 and allows an extra period of assessment in how the software performs.
 
-The stable-{product-version} channel also allows upgrading from a previous minor version of
+The `stable-{product-version}` channel also allows upgrading from a previous minor version of
 {product-title}.
 
 [discrete]
-== Upgrade Version Paths
+== Upgrade version paths
 
 {product-title} maintains an upgrade recommendation service that understands the
 version of {product-title} you have installed as well as the path to take within
 the channel you choose to get you to the next release. You can imagine seeing the
-following in the fast-{product-version} channel:
+following in the `fast-{product-version}` channel:
 
 * {product-version}.0
 * {product-version}.1
 * {product-version}.3
 * {product-version}.4
 
-The service only recommends upgrades that have been tested and have no serious issues.
-If you are on {product-version}.1 and {product-title} is allowing you to select
-{product-version}.4, then it is safe for you to go from .{product-version}.1 to .{product-version}.4. Likewise,
-the absence of {product-version}.2 may be due to a CVE that was fixed in {product-version}.3 and Red Hat no
-longer suggests upgrading to a known vulnerable version.
+The service recommends only upgrades that have been tested and have no serious issues.
+If your cluster is on {product-version}.1 and {product-title} suggests
+{product-version}.4, then it is safe for you to update from .{product-version}.1 to .{product-version}.4. If {product-version}.2 is no longer available from the update service, it might be due to a vulnerability that was fixed in {product-version}.3. The update service will not suggest updating to a version of {product-title{ that contains known vulnerabilities.
```

Comment:
"no longer available" implies that the release was in the channel at some point, but we never remove releases from channels, we just block incoming edges.  Sometimes we cut a release and then decide before it ever lands in a channel (even candidate) that it is too buggy to ship.  An example is [4.3.4][1].  We also skip patch numbers if we go more than a week between releases (e.g. because we have no bugs to fix).  An example is [4.1.32][2].  Can we update this wording to be closer to:

> Do not rely on consecutive patch numbers.  For example, {product-version}.2 is not, and never was, available in the example above.

And not get into why we sometimes skip over patches?

[1]: https://github.com/openshift/cincinnati-graph-data/blob/61f8a42332b767613859535bd53c22ba24b6665c/channels/candidate-4.3.yaml#L24
[2]: https://github.com/openshift/cincinnati-graph-data/blob/61f8a42332b767613859535bd53c22ba24b6665c/channels/prerelease-4.1.yaml#L40

Reply:
@wking, I am happy to not get into why we skip patches.

Reply:
> I am happy to not get into why we skip patches.

Right, but I still think we still need to say something like my above proposal to mention that release versions may not be consecutive without getting into why.